ci/docker/python-wheel-windows-vs2022.dockerfile

```diff
@@ -29,7 +29,15 @@ RUN (if "%python%"=="3.10" setx PYTHON_VERSION "3.10.11" && setx PYTHON_CMD "py
     (if "%python%"=="3.13" setx PYTHON_VERSION "3.13.7" && setx PYTHON_CMD "py -3.13") & \
     (if "%python%"=="3.14" setx PYTHON_VERSION "3.14.0" && setx PYTHON_CMD "py -3.14")
 
-RUN choco install -r -y --pre --no-progress python --version=%PYTHON_VERSION%
```

Comment:
I'm not sure we still need the `PYTHON_VERSION` selection above. It's probably ok to run `pymanager install --version 3.13 ...`, for example.